### DISCUSSION TITLE
Bug:  The Youtube API key you provided was not a string

### DISCUSSION THREAD

**Discussion by BlueMaxWasTaken** [eritislami/evobot#296] [Q&A]
I have that error when i run the bot. how can i fix it?

**Comment by wolf-yuan-6115** [comment]
Can you send your `config.json` code ?
Please hide your bot's token and YouTube API key

**Reply by unofficialdxnny** [reply]
can you please help me. basically i just added some more code to my bot ( a sound board) but now i run the bot it says :Error: The YouTube API key you provided was not a string.

<CONFIG CODE>
  "TOKEN": "BOT TOKEN",
  "YOUTUBE_API_KEY": "YT KEY",
  "SOUNDCLOUD_CLIENT_ID": "",
   "Spotify_ClientID": "",
  "Spotify_ClientSecret": "",
  "MAX_PLAYLIST_SIZE": 1000,
  "PREFIX": "-",
  "PRUNING": false,
  "LOCALE": "en",
  "STAY_TIME": 1209600,
  "DEFAULT_VOLUME": 100

**Comment by eritislami** [comment]
Not a bug, converting to discussion

**Reply by wolf-yuan-6115** [reply]
Eww, he is leaved
